cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,103 @@
 /// <reference types='cypress' />
+const { generateWorker } = require('../support/generateWorker');
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let worker;
 
+  beforeEach(() => {
+    cy.task('generateWorker').then((generateWorker) => {
+      worker = generateWorker;
+      cy.visit('/');
+    });
+  });
+
+  it('should check pagination', () => {
```

Comment:
add test for clicking next and previous buttons

Reply:
OK